Update group-update.md (#9637)
microsoftgraph/microsoft-graph-docs-contrib: api-reference/v1.0/api/group-update.md

| visibility              | String  | Specifies the visibility of a Microsoft 365 group. The possible values are: **Private**, **Public**, or empty (which is interpreted as **Public**).                                                                                                                                                                                                                                                                                                                                          |

> [!IMPORTANT]

> > - To update the following properties, you must specify them in their own PATCH request, without including the other properties listed in the previous table: **allowExternalSenders**, **autoSubscribeNewMembers**, **hideFromAddressLists**, **hideFromOutlookClients**, **isSubscribedByMail**, **unseenCount** and must have **Group.ReadWrite.All** permission to update any of these properties

Reviewer: ```suggestion
> - To update these properties (**allowExternalSenders**, **autoSubscribeNewMembers**, **hideFromAddressLists**, **hideFromOutlookClients**, **isSubscribedByMail**, **unseenCount**), you must:
>    - Specify them in their own PATCH request without including other properties from the previous table
>    - Have the Group.ReadWrite.All permission (*Directory.ReadWrite.All* is not supported for these properties)
```
